demo/package.json

```diff
@@ -12,8 +12,12 @@
     "bluebird": "2.9.34",
     "body-parser": "1.13.3",
     "connect": "3.4.0",
+    "connect-chain": "0.0.1",
+    "connect-query": "^1.0.0",
```

Comment:
Please lock down version numbers (I know these are automatically locked but just in case the lock file needs to be recreated)